plugins/talk-plugin-moderation-actions/client/actions.js

```diff
@@ -8,3 +8,12 @@ export const openMenu = (id) => ({
 export const closeMenu = () => ({
   type: CLOSE_MENU,
 });
+
+export const openDialog = (comment) => ({
```

Comment:
I would avoid passing too much data. Moreover `comment` is always a fragment and the content may vary.
Changing this to sth like `commentId, authorId` makes the data requirement clear.

Reply:
Hm, while I was doing this I realized this change does not look good. I also need `commentStatus` and it's a lot to rename. The amount of data passed through `comment` is being handled by the `fragment` so I don't think this change is needed.

Reply:
By looking at this action in isolation it's not defined what data I need to pass to `comment`. Only if I have a good knowledge of the whole plugin, I'd know that it needs the comment id the author id and the comment status and it must follow graph schema. Under the condition that this action is not used anywhere else outside of the plugin (e.g. someone wants to trigger the ban dialog from a different plugin) than we could argue that it's fine. My recommendation is to have clear data requirements and only pass what is necessary (right now `tags` are also passed to redux but it is not used).